test/rest-coercion/urlencoded-date.suite.js

```diff
@@ -0,0 +1,76 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
+// Node module: strong-remoting
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+var urlEncodedContext = require('./_urlencoded.context');
+var builders = require('../helpers/builders');
+
+var remotingDate = builders.remotingDate;
+var jsonDate = builders.jsonDate;
+var INVALID_DATE = builders.REMOTING_DATE_INVALID;
+
+module.exports = function(ctx) {
+  suite('query string', urlEncodedContext(ctx, 'qs'));
+  suite('form data', urlEncodedContext(ctx, 'form'));
+};
+
+function suite(prefix, ctx) {
+  var EMPTY_QUERY = ctx.EMPTY_QUERY;
+  var ERROR_BAD_REQUEST = ctx.ERROR_BAD_REQUEST;
+  var verifyTestCases = ctx.verifyTestCases;
+
+  describe('query string - date - required', function() {
+    verifyTestCases({ arg: 'arg', type: 'date', required: true }, [
+      [EMPTY_QUERY, ERROR_BAD_REQUEST],
+      ['arg', INVALID_DATE], // should be: ERROR_BAD_REQUEST
+      ['arg=', INVALID_DATE], // should be: ERROR_BAD_REQUEST
+      ['arg=undefined', INVALID_DATE], // should be: ERROR_BAD_REQUEST
+      ['arg=null', INVALID_DATE], // should be: ERROR_BAD_REQUEST
+      ['arg=2016-05-19T13:28:51.299Z', remotingDate('2016-05-19T13:28:51.299Z')],
+    ]);
+  });
+
+  describe('query string - date - optional', function() {
+    verifyTestCases({ arg: 'arg', type: 'date' }, [
+      [EMPTY_QUERY, undefined],
```

Comment:
Just a nit: its hard to tell what the eventual assertions are here... a comment would be enough.